src/replica/replica.cpp

```diff
@@ -229,11 +228,7 @@ void replica::update_last_checkpoint_generate_time()
 //            //
 // Statistics //
 //            //
-
-void replica::update_commit_qps(int count)
-{
-    _stub->_counter_replicas_commit_qps->add((uint64_t)count);
-}
+METRIC_IMPL_INCREMENT_BY(*_stub, committed_requests, replica::)
```

Comment:
It‘s a bit of strange to pass a namesapce as a "parameter", would it better to define functions manually?